compact/definitions.xml

```diff
@@ -514,17 +515,18 @@ Service gaps in FW direction (before endcapP ECAL) and BW direction (before endc
     <comment> extra 50cm rmax that "protrudes" into the HCAL</comment>
     <constant name="EcalEndcapP_rmax"               value="floor(Eta1_1_tan * EcalEndcapP_zmin)"/>
 
-    <constant name="EcalEndcapN_zmin"               value="BackwardPIDRegion_zmin + BackwardInnerEndcapRegion_length"/>
-    <constant name="EcalEndcapN_length"             value="60*cm" />
+    <!-- <constant name="EcalEndcapN_zmin"               value="BackwardPIDRegion_zmin + BackwardInnerEndcapRegion_length"/> -->
+    <constant name="EcalEndcapN_zmin"               value="174*cm"/>   <!-- Currently fix value hardcoded -->
+    <constant name="EcalEndcapN_length"             value="30*cm" />   <!-- Currently fix value hardcoded -->
     <constant name="EcalEndcapN_zmax"               value="EcalEndcapN_zmin + EcalEndcapN_length"/>
     <comment>
       rmin1: rmin round electron pipe (ignoring the hadron pipe)
       rmin2: rmin around both beam pipes
     </comment>
     <constant name="EcalEndcapN_rmin1"              value="Eta4_6_tan * EcalEndcapN_zmin"/>
     <constant name="EcalEndcapN_rmin2"              value="Eta3_9_tan * EcalEndcapN_zmin"/>
-    <constant name="EcalEndcapN_rmin"               value="max(EcalEndcapN_rmin1, EcalEndcapN_rmin2)"/>
-    <constant name="EcalEndcapN_rmax"               value="63*cm"/> <!-- FIXME hardcoded -->
+    <constant name="EcalEndcapN_rmin"               value="8.5*cm"/>   
+    <constant name="EcalEndcapN_rmax"               value="64.1*cm"/>  
```

Comment:
These are not in agreement with https://eic.jlab.org/Geometry/Detector/Detector-20220929172703.html. Please don't use hardcoded numbers in here; they should come from `definitions.xml` where the master parametrization is.

Reply:
Never mind, this is `definitions.xml`. Please put the correct values here, which are from the detector geometry database.

Reply:
These number I will check with Carlos, then I will move those numbers to the endcap.xml file

Reply:
OK, got it. I will check the number

Reply:
Here I put the correct number in the last commit